### DISCUSSION TITLE
Client credentials grant

### DISCUSSION THREAD

**Discussion by Pavel-Husakouski** [panva/openid-client#491] [Q&A] [answered]
Is there a documentation or test how to implement the client credentials grant using the `openid-client` package?
The readme clearly says, that the package does support this grant, however, I can't understand what code to write.

**Comment by panva** [comment, marked as answer]
https://github.com/panva/node-openid-client/blob/v5.1.6/docs/README.md#clientgrantbody-extras

**Reply by Pavel-Husakouski** [reply]
Thanks for the answer! 
Is there a way to verify the jwt token returned by grant method using this library using some public key or jwks endpoint?

**Reply by panva** [reply]
This is a client library, not a resource server library, therefore, verifying Access Tokens is out of scope.
